<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSix</class>
 <widget class="QWidget" name="WidgetSix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="WidgetASI" name="widgetASI" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>99</horstretch>
         <verstretch>99</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="WidgetAI" name="widgetAI" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>99</horstretch>
         <verstretch>99</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="WidgetALT" name="widgetALT" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>99</horstretch>
         <verstretch>99</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="WidgetTC" name="widgetTC" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>99</horstretch>
         <verstretch>99</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="WidgetHI" name="widgetHI" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>99</horstretch>
         <verstretch>99</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="WidgetVSI" name="widgetVSI" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>99</horstretch>
         <verstretch>99</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetASI</class>
   <extends>QWidget</extends>
   <header>example/WidgetASI.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetALT</class>
   <extends>QWidget</extends>
   <header>example/WidgetALT.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTC</class>
   <extends>QWidget</extends>
   <header>example/WidgetTC.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetVSI</class>
   <extends>QWidget</extends>
   <header>example/WidgetVSI.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetAI</class>
   <extends>QWidget</extends>
   <header>example/WidgetAI.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetHI</class>
   <extends>QWidget</extends>
   <header>example/WidgetHI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
